arbitrum/uniswap/eip712.json

```diff
@@ -80,6 +80,451 @@
                             }
                         ]
                     }
+                },
+                {
+                    "mapper": {
+                        "fields": [
+                            {
+                                "assetPath": "permitted.token",
+                                "format": "amount",
+                                "label": "Approve amount",
+                                "path": "permitted.amount"
+                            },
+                            {
+                                "assetPath": "witness.inputToken",
+                                "format": "amount",
+                                "label": "To swap",
+                                "path": "witness.inputAmount"
+                            },
+                            {
+                                "assetPath": "witness.outputs[0].token",
+                                "format": "amount",
+                                "label": "Receive at Minimum",
+                                "path": "witness.outputs[0].amount"
+                            },
+                            {
+                                "label": "On Address",
+                                "path": "witness.outputs[0].recipient"
+                            }
```

Comment:
These fields need to be shown to the user too ?
```
                            {
                                "label": "Spender",
                                "path": "spender"
                            },
                            {
                                "format": "datetime",
                                "label": "Deadline",
                                "path": "deadline"
                            }
```

Reply:
change added